app/controllers/authors_controller.rb

```diff
@@ -0,0 +1,51 @@
+class AuthorsController < ApplicationController
+  layout 'authors_header'
+
+  def index
+    @authors = Author.all
+  end
+
+  def show
+    @author = Author.find(params[:id])
```

Comment:
Create one method for finding a author and use the same method every where, using before_action callback.